lib/components/primary_button.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_movie_clean/gen/colors.gen.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
+class PrimaryButton extends StatelessWidget {
+  const PrimaryButton(
+      {this.width,
+      this.height,
+      this.onPressed,
+      this.title,
+      this.textStyle,
+      super.key});
+
+  final double? width;
+  final double? height;
+  final Function? onPressed;
+  final String? title;
+  final TextStyle? textStyle;
+
+  @override
+  Widget build(BuildContext context) {
+    final size = MediaQuery.of(context).size;
+    return MaterialButton(
+      color: AppColors.crimsonApprox,
+      height: height ?? 50.h,
+      child: SizedBox(
+        width: width ?? size.width,
+        child: Text(
+          title ?? '',
+          textAlign: TextAlign.center,
+          style: TextStyle(
+              fontSize: 18.38.sp,
+              color: AppColors.white,
+              fontWeight: FontWeight.w600),
+        ),
+      ),
+      onPressed: () {
+        onPressed?.call();
+      },
```

Comment:
just a suggestion, u can skip this
```suggestion
      onPressed: () => onPressed?.call(),
```